Global.h

```diff
@@ -698,3 +698,16 @@ namespace vstd
 }
 using vstd::operator-=;
 using vstd::make_unique;
+
+#if defined VCMI_ANDROID && defined NO_STD_TOSTRING

```

Comment:
Well, I can add this define, but that is a hack as well. 
But yeah, as long as it's only to_string, it's not a big problem to have our impl. As far as I can see, the only methods guarded by this in basic_string header are to_string and various string->number conversions.